model/svg_info.go

```diff
@@ -0,0 +1,18 @@
+package model
+
+import (
+	"encoding/xml"
+)
+
+type SVGInfo struct {
```

Comment:
Thinking we probably don't need to export `SVGInfo` in the `model` package -- these structs are meant for public consumption (e.g. plugins compiling against this package), but I don't think anyone but `UploadFileX` needs this one.

I propose inlining this into a helper method in the `app` package (requiring no `model` changes at all), e.g.:
```go
func parseSVGAttributes(fileInfo *model.FileInfo, svgData []byte) {
    var svgInfo struct {
        // ...
    }

    if err := xml.Unmarshal(svgData, &svgInfo); err != nil {
        mlog.Error(...)
    }

    // ...
}
```